### PR TITLE
[4.0] incorrect terminology

### PR DESCRIPTION
**Pull request by brianteeman**
PR for #19725

It is NOT a database table name you are entering - it is the database name.

and replace specify with enter at the same time (plain english)

### CODE REVIEW

**Comment by astridx** [comment]
@brianteeman Your branch is out-of-date. Can you merge the latest changes, please.
I get an error when I tested. That error was fixed recently (https://github.com/joomla/joomla-cms/pull/19689).

File: installation/language/en-GB/en-GB.ini [hunk 1]
```diff
@@ -29,16 +29,16 @@ INSTL_PRECHECK_ACTUAL="Actual"
...
-INSTL_DATABASE_HOST_DESC="Specify the Host Name, usually \"localhost\" or a name provided by your host."
+INSTL_DATABASE_HOST_DESC="Enter the Host Name, usually \"localhost\" or a name provided by your host."
 INSTL_DATABASE_HOST_LABEL="Host Name"
 INSTL_DATABASE_HOST_IS_NOT_LOCALHOST_CREATE_FILE="We were not able to create the file. Please manually create a file named &quot;%1$s&quot; and upload it to the &quot;%2$s&quot; folder of your Joomla site."
 INSTL_DATABASE_HOST_IS_NOT_LOCALHOST_DELETE_FILE="In order to confirm that you are the owner of this website please delete the file named &quot;%1$s&quot; we have created in the &quot;%2$s&quot; folder of your Joomla site."
 INSTL_DATABASE_HOST_IS_NOT_LOCALHOST_GENERAL_MESSAGE="You are trying to use a database host which is not on your local server. For security reasons, you need to verify the ownership of your web hosting account. <a href="_QQ_"%s"_QQ_">Please read the documentation</a> for more information."
-INSTL_DATABASE_NAME_DESC="Specify the database table name. Use table prefix to distinct Joomla sites."
+INSTL_DATABASE_NAME_DESC="Enter the database table name."
```
Comment (Quy) [inline installation/language/en-GB/en-GB.ini]: `database name`

Comment (brianteeman) [reply installation/language/en-GB/en-GB.ini]: less haste more speed

File: installation/language/en-GB/en-GB.ini [hunk 2]
```diff
@@ -29,16 +29,16 @@ INSTL_PRECHECK_ACTUAL="Actual"
 ; Database view
 INSTL_DATABASE="Database Configuration"
 INSTL_DATABASE_ERROR_POSTGRESQL_QUERY="PostgreSQL database query failed."
-INSTL_DATABASE_HOST_DESC="Specify the Host Name, usually \"localhost\" or a name provided by your host."
+INSTL_DATABASE_HOST_DESC="Enter the Host Name, usually \"localhost\" or a name provided by your host."
```
Comment (Quy) [inline installation/language/en-GB/en-GB.ini:32]: lower case `Host Name`? This is the case with `database name` below.

Comment (brianteeman) [reply installation/language/en-GB/en-GB.ini:32]: no it needs to be caps to differentiate from "name provided by your host"

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 35219ecec47c1b0c24b7223e31e3a1d190d7e0d5<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19728">issues.joomla.org/tracker/joomla-cms/19728</a>.</sub>

**Comment by dgrammatiko** [comment]
I have tested this item :white_check_mark: successfully on 35219ecec47c1b0c24b7223e31e3a1d190d7e0d5<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19728">issues.joomla.org/tracker/joomla-cms/19728</a>.</sub>

**Comment by parul-123** [comment]
I have tested this item :white_check_mark: successfully on 35219ecec47c1b0c24b7223e31e3a1d190d7e0d5<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19728">issues.joomla.org/tracker/joomla-cms/19728</a>.</sub>

**Comment by samarthsharma36** [comment]
I have tested this item :white_check_mark: successfully on 0f78edab6238fa326fa8145a2b235b8c354e74a7<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19728">issues.joomla.org/tracker/joomla-cms/19728</a>.</sub>

**Comment by zero-24** [comment]
cc @wilsonge @laoneo

**Pull request merged**